### ISSUE TITLE
Can we add the output logits of wide and deep directly?

### ISSUE THREAD

**Issue by chenhui-bupt** [ichuang/tflearn_wide_and_deep#3]
I wonder if the output logits of deep model has different magnitude with the wide model, can we add them directly as our prediction value? can someone tell whether we should pay attention to this problem or not, and how to solve it, thanks very much.
